Code/JavaWrappers/gmwrapper/CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ ADD_CUSTOM_COMMAND(
   COMMAND ${CMAKE_COMMAND} -E make_directory build-test
 
   ## Add in our own Java sources
-  #COMMAND ${COPY_CMD}
+  COMMAND ${COPY_CMD}
```

Comment:
This also copies over Lipinski.java, any idea why this was commented out or if it should be copied over again?